presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ControlAPI.java

```diff
@@ -39,13 +39,7 @@
 @Singleton
 public class ControlAPI
 {
-    private PrestoRpmController controller;
-
-    @Inject
-    public ControlAPI(PrestoRpmController controller)
-    {
-        this.controller = controller;
-    }
+    private static final PrestoController CONTROLLER = new RpmManager();
```

Comment:
The control API class shouldn't know how it is implemented. That's why the general name.